tensorflow/python/keras/mixed_precision/experimental/loss_scale.py

```diff
@@ -79,7 +79,8 @@ def update(self, grads):
     Args:
       grads: A list of unscaled gradients, each which is the gradient of the
         loss with respect to a weight. The gradients should have already been
-        divided by the loss scale being before passed to this function.
+        divided by the loss scale being before passed to this function. 'None'
+        gradients are accepted, and should be ignored.
```

Comment:
Replace "should be" with "are". And same for the other LossScale class.

Reply:
Done